src/nginxconfig/generators/conf/website.conf.js

```diff
@@ -136,6 +136,18 @@ export default (domain, domains, global) => {
         serverConfig.push(...securityConf(domains, global));
     }
 
+    // Restrict Methods
+    if (Object.keys(domain.restrict).find(k => domain.restrict[k].computed)) {
+        const allowedKeys = Object.keys(domain.restrict)
+                                .filter(k => !domain.restrict[k].computed)
+                                .map(e => e.replace('Method', '').toUpperCase());
+
+        serverConfig.push(['# restrict Methods', '']);
+        serverConfig.push(['location /', {
+            limit_except: `${allowedKeys.join(' ')} { deny all; }`,
+        }]);
```

Comment:
I don't think this should be in a location block? I think it can just be left bare directly in the server block so that it applies to everything? Having it in its own location block will break other location blocks iirc, as nginx only runs the first matching location block?
```suggestion
        serverConfig.push(['limit_except', `${allowedKeys.join(' ')} { deny all; }`]);
```

Reply:
Hey! I'm definitely not an nginx expert, so I've honestly got very little idea on the best practice here. Doing some research, it looks like folks recommend using an if statement instead of limit_except if its needed across the entire server block: https://serverfault.com/questions/1011021/limit-except-vs-request-method-getheadpost

Reply:
Ok Matt, I've changed the implementation to an If statement, and tested it. Works as intended.